exporter.go

```diff
@@ -93,6 +94,11 @@ func (cc *ConohaCollector) AutoUpdate() {
 			metrics = append(metrics, prometheus.MustNewConstMetric(cc.describes[7], prometheus.GaugeValue, float64(quota.TotalUsage), serviceID))
 		}
 
+		deposit, err := cc.BillingPaymentSummary()
+		if err != nil {
+			log.Fatal(err)
+		}
+		metrics = append(metrics, prometheus.MustNewConstMetric(cc.describes[8], prometheus.GaugeValue, float64(deposit.Deposit), "deposit"))
```

Comment:
```suggestion
		metrics = append(metrics, prometheus.MustNewConstMetric(cc.describes[8], prometheus.GaugeValue, float64(deposit.Deposit)))
```
31行目のコメント(`// NewDescの3番目の引数は可変ラベル（NewConstMetricの最後の可変長引数に対応してる）`)
的にここはいらなさそうです

Reply:
そこと対応しているんですね、なるほど。